Makefile

```diff
@@ -191,6 +191,9 @@ endif
 upb_json_SRCS = \
   upb/json/parser.c \
   upb/json/printer.c \
+  upb/pb/encoder.c \
```

Comment:
We shouldn't put these sources in more than one library. Instead add dependencies on libupb.pb.a to binaries that need it.

Or help me get Kokoro Bazel tests going, so we can delete the Travis tests and the Makefile completely.

Reply:
Left this. Will fix it after kokoro fix.